config/npi_probing_finetune.conf

```diff
@@ -0,0 +1,95 @@
+// Config modifications to apply during all final *BERT* runs.
+
+// This imports the defaults, which can be overridden below.
+include "defaults.conf"
```

Comment:
This'll be much easier to debug if you only put new content here. If this is based on one of the other conf files, include that instead of defaults, and only include lines you change.